packages/smooth_app/lib/pages/preferences/user_preferences_list_tile.dart

```diff
@@ -30,22 +30,43 @@ class UserPreferencesListTile extends StatelessWidget {
             ? MainAxisAlignment.start
             : MainAxisAlignment.spaceBetween,
         crossAxisAlignment: CrossAxisAlignment.center,
-        children: <Widget>[title, icon!],
+        children: <Widget>[
+          title,
+          icon!,
+        ],
       );
     }
+
+    final TextStyle textStyle = DefaultTextStyle.of(context).style;
+
     final Widget displayed = Padding(
       padding: const EdgeInsets.symmetric(
         horizontal: LARGE_SPACE,
-        vertical: SMALL_SPACE,
+        vertical: LARGE_SPACE,
       ),
       child: Column(
         mainAxisAlignment: MainAxisAlignment.start,
         crossAxisAlignment: CrossAxisAlignment.start,
         children: <Widget>[
           const SizedBox(height: SMALL_SPACE),
-          tmpTitle,
+          DefaultTextStyle.merge(
+            child: tmpTitle,
+            style: Theme.of(context).textTheme.headline4,
+          ),
           const SizedBox(height: VERY_SMALL_SPACE),
-          if (subtitle != null) subtitle!,
+          if (subtitle != null)
+            ConstrainedBox(
+              // At least two lines for the subtitle
+              constraints: BoxConstraints(
+                minHeight: (textStyle.fontSize! * 2) + 1.5 * 2,
```

Comment:
What is the `+ 1,5 * 2` for

Reply:
*2 : for 2 lines
1,5 is the line height